src/Control/Monad/Trans/Uncertain.hs

```diff
@@ -1,15 +1,15 @@
-{-# LANGUAGE PackageImports, RankNTypes #-}
+{-# LANGUAGE PackageImports #-}
+{-# LANGUAGE RankNTypes     #-}
 -- | A computation which may raise warnings or fail in error.
 module Control.Monad.Trans.Uncertain where
 
-import Control.Applicative
-import "mtl" Control.Monad.Trans
-import "mtl" Control.Monad.Identity
-import "transformers" Control.Monad.Trans.Error hiding (Error)
-import "transformers" Control.Monad.Trans.Writer
-import System.Exit
-import System.IO
-import Text.Printf
+import           "mtl" Control.Monad.Identity
+import           "mtl" Control.Monad.Trans
+import           "transformers" Control.Monad.Trans.Error  hiding (Error)
+import           "transformers" Control.Monad.Trans.Writer
```

Comment:
Ah! Now I can see that when there is more than one import from the same explicitly-named package, the imports within a package are aligned. So this generated alignment is fine (except for the gap between the module name and the `as` or `hiding` keyword, as previously discussed)

As a side node, I wonder why I'm importing both `"mtl"` and `"transformers"` here? Usually one picks one style or the other, not a mix of them. I personally prefer the `"transformers"` style (not a very popular choice, I know...), but I'm not the only person who touched this project.